### ISSUE TITLE
TUNNELCOMM Messages

### ISSUE THREAD

**Issue by marshall-davis** [skotostech/websocket-to-tcp-tunnel#8]
When the games can accept such messages we should use this to send informative updates.

**Comment by noahgibbs** [comment]
Do you mean the current TUNNELINFO messages (src/Relay.js line 110)? It *is* annoying that we have that there and it's clearly meant to communicate with the SkotOS server but it's turned off because SkotOS can't read or parse them yet :-/

**Comment by marshall-davis** [comment]
Yeah, I may have changed or described it incorrectly but I just meant the out of band information that is more meta data about the connection rather than data passing through the connection.

When I said "when the games can accept" such I meant whatever system managers said game really, be that internal or external to the game itself.

**Comment by noahgibbs** [comment]
That makes sense. Right now the TUNNELINFO messages could be used to detect disconnect/reconnect and include the headers set on the connection, which can sometimes be useful. But yeah, right now ChatTheatre/SkotOS (and SkotOS in general, I think) doesn't parse or use those messages, and they're turned off in production.

**Comment by marshall-davis** [comment]
In theory they could also be useful in the other direction. A client could make use of them as well. I don't remember if the client I started did or not, but I think it tried to send and parse some.

That being said, I've not touched this for so long if you wanted to clean house and close this ancient issue I wouldn't take offense.

**Comment by noahgibbs** [comment]
Currently the tunnel only sends things the client can already see -- headers which are sent by the client, plus connection/disconnection. So currently the client wouldn't get any new information from them.
